pkg/skaffold/schema/profiles_test.go

```diff
@@ -464,6 +464,49 @@ func TestActivatedProfiles(t *testing.T) {
 	}
 }
 
+func TestYamlAlias(t *testing.T) {
+	config := `
+.activation_common: &activation_common
+  env: ABC=common
+build:
+  artifacts:
+  - image: example
+profiles:
+- name: simple1
+  activation:
+  - *activation_common
+  - env: ABC=1
+- name: simple2
+  activation:
+  - *activation_common
+  - env: ABC=2
+`
+
+	testutil.Run(t, "", func(t *testutil.T) {
+		tmpDir := t.NewTempDir().
+			Write("skaffold.yaml", addVersion(config))
+
+		parsed, err := ParseConfig(tmpDir.Path("skaffold.yaml"), false)
+		t.CheckNoError(err)
+		if parsed == nil || err != nil {
+			t.Fail()
+			return
+		}
+
+		skaffoldConfig := parsed.(*latest.SkaffoldConfig)
+
+		t.CheckDeepEqual([]latest.Profile{
+			{Name: "simple1", Activation: []latest.Activation{{Env: "ABC=common"}, {Env: "ABC=1"}}},
+			{Name: "simple2", Activation: []latest.Activation{{Env: "ABC=common"}, {Env: "ABC=2"}}},
+		}, skaffoldConfig.Profiles)
+
+		err = ApplyProfiles(skaffoldConfig, cfg.SkaffoldOptions{
+			Profiles: []string{"simple1"},
+		})
+		t.CheckNoError(err)
```

Comment:
Should you add assertions on the resulting config?

Reply:
Done